reginline.h

```diff
@@ -14,7 +14,7 @@ Perl_regnext(pTHX_ const regnode *p)
         return(NULL);
 
     if (OP(p) > REGNODE_MAX) {                /* regnode.type is unsigned */
-        Perl_croak(aTHX_ "Corrupted regexp opcode %d > %d",
```

Comment:
panic

Reply:
That sounds like a good suggestion for another branch. In this branch I'm aiming not to have any behavioral changes.